acceptance/suites/pre_suite/foss/30_install_dev_repos.rb

```diff
@@ -4,6 +4,8 @@
     package_build_version = ENV['PACKAGE_BUILD_VERSION']
     if package_build_version
       install_puppetlabs_dev_repo master, 'puppetserver', package_build_version
+      # Cleanup the local repo copy to avoid problems
+      FileUtils.rm_rf('tmp/repo_configs') if File.exists?('tmp/repo_configs')
```

Comment:
I think we would either want to delete tmp/repo_configs/<variant> (see [beaker/install_utils.rb#969](https://github.com/puppetlabs/beaker/blob/1.x/lib/beaker/dsl/install_utils.rb#L969)) or pass in a custom destination dir.